src/token/token.service.ts

```diff
@@ -1,11 +1,32 @@
 import { Injectable, Logger } from '@nestjs/common';
-import { Cron } from '@nestjs/schedule';
+import { Cron, CronExpression } from '@nestjs/schedule';
 import { InjectRepository } from '@nestjs/typeorm';
 import axios from 'axios';
 import { Repository } from 'typeorm';
 
 import { Token } from './token.entity';
 import { configService } from '../config.service';
+import initializeNearService from 'src/nearService';
```

Comment:
Please always use relative paths for local imports to avoid issues with building in various context and tests.